app/content/serializers/event.py

```diff
@@ -129,9 +129,10 @@ class Meta:
         )
 
     def create(self, validated_data):
-        event = Event.objects.create(**validated_data)
         if "registration_priorities" in validated_data:
             registration_priorities_data = validated_data.pop("registration_priorities")
+        event = Event.objects.create(**validated_data)
```

Comment:
Please call `super().create(validated_data)` here so the action is logged as well 😄